spec/request_handler_spec.rb

```diff
@@ -156,5 +159,39 @@
         end
       end
     end
+
+    context "when request name = 'tracking_by_consignment'" do
+      context "with valid parcel number" do
+        it "returns successful response" do
+          VCR.use_cassette('tracking_by_consignment_with_valid_parcel_number') do
+            response = handler.request(:tracking_by_consignment, { parcel_number: valid_parcel_number })
+            expect(response).to_not be_nil
```

Comment:
I think we should be clearer in our spec expectations - 'nil' is a bit ambiguous when we expect something specific. The response in the fixture shows response content that can be tested here.